### ISSUE TITLE
tensorflowjs_converter command not found in Ubuntu

### ISSUE THREAD

**Issue by jtiscione** [tensorflow/tfjs#2612]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version

tensorflow-gpu 2.0.0

#### Describe the problem or feature request

Tensorflowjs_converter is either not being installed with Tensorflow or isn't installing correctly (Ubuntu 18.04, tensorflow-gpu 2.0.0).

#### Code to reproduce the bug / link to feature request
If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

Installing according to [official instructions](https://www.tensorflow.org/install/pip?lang=python3):

```
conda create -n tf2 pip python=3.7
conda activate tf2
pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow_gpu-2.0.0-cp37-cp37m-manylinux2010_x86_64.whl
tensorflowjs_converter
```

Prints
`tensorflowjs_converter: command not found`

Running pip with sudo makes no difference.

**Comment by oanush** [comment]
@jtiscione ,
Cab you please check the solutions provided in this [link](https://stackoverflow.com/questions/50317081/tensorflowjs-converter-command-not-found) and let us know if it helps resolving the issue? Thanks!

**Comment by jtiscione** [comment]
Never mind; I was thinking tensorflowjs was included in the 2.0 wheel but it has Tensorflow 1.15 as a dependency.
